.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+packages
```

Comment:
I am not a huge fan of this pattern, as one of the strengths of monorepos in my experience is the colocation of `devDependencies` and unification of their execution. This repo is a variant on the "classic" monorepo structure, of course, but I don't think it really demands repetitious eslint config and scripts in every subdir of `packages`.